Update ReadMe (#1538)
microsoft/WinUI-Gallery: README.md

# WinUI 3 Gallery 

[![Build Status](https://dev.azure.com/ms/microsoft-ui-xaml/_apis/build/status%2Fmicrosoft.WinUI-Gallery?branchName=main)](https://dev.azure.com/ms/microsoft-ui-xaml/_build/latest?definitionId=612&branchName=main) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com) [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg?style=flat-square)](https://github.com/your/your-project/blob/master/LICENSE)

![WinUI Screenshot](README_Images/WinUIGalleryLanding.PNG)

### Accompanying Sample App for [WinAppSDK](https://github.com/microsoft/WindowsAppSDK)

> This app demonstrates all of the Windows UI 3 library controls and styles available to make a WinUI 3 app with the [Windows App SDK](https://github.com/microsoft/WindowsAppSDK). It is the interactive companion to the [Fluent Design Guidelines](https://docs.microsoft.com/windows/apps/design/basics/) and shows the usage of the [WinUI](https://docs.microsoft.com/windows/apps/winui/) APIs. 

Reviewer: ```suggestion
This app demonstrates all of the WinUI 3 controls and styles available to make a modern Windows app with the [Windows App SDK](https://github.com/microsoft/WindowsAppSDK). It is the interactive companion to the [Fluent Design Guidelines](https://docs.microsoft.com/windows/apps/design/basics/) and shows the usage of the [WinUI](https://docs.microsoft.com/windows/apps/winui/) APIs. 

```

## 🚀 Getting Started
Quick start guide to building WinUI 3 Gallery locally:

### 1. Set up the Environment

>**Note**: The WinUI 3 Gallery requires [Visual Studio 2022](https://visualstudio.microsoft.com/vs/) or later to build and Windows 10 or later to execute.
If you're building an app with the Windows App SDK for the first time, follow the [installation instructions here.](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/set-up-your-development-environment)

**Required [Visual Studio components](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/set-up-your-development-environment?tabs=cs-vs-community%2Ccpp-vs-community%2Cvs-2022-17-1-a%2Cvs-2022-17-1-b#required-workloads-and-components):**
- .NET Desktop Development.
- Windows App SDK C# Templates
- Windows 10 SDK (10.0.19041.0)

### 2. Clone the Repository

```shell
git clone https://github.com/microsoft/WinUI-Gallery.git
```

### 3. Open .sln with Visual Studio and Build!
```shell
.\WinUIGallery\WinUIGallery.DesktopWap.sln
```
Make sure to select `WinUIGallery.DesktopWap.Package` for startup project in Visual Studio.

## ➡️ Further information

> To obtain information about Windows 11 development, go to the [Windows Dev Center](https://developer.microsoft.com/windows)  

Reviewer: ```suggestion
To obtain information about WinUI development, go to the [Windows Dev Center](https://developer.microsoft.com/windows)  

```

> To obtain information about Microsoft Visual Studio and the tools for developing Windows apps, go to [Visual Studio](http://go.microsoft.com/fwlink/?LinkID=532422)  

Reviewer: Is this VS release notes page useful?

Author: You're right, let me remove it.

To contact the authors, please reach out to ControlsGallery@microsoft.com

### Related topics

> [Get started with Windows 11 apps](https://docs.microsoft.com/windows/apps/get-started)  

Reviewer: ```suggestion
[Get started with building WinUI apps](https://learn.microsoft.com/en-us/windows/apps/get-started/start-here)  

```

Reviewer 2: I don't think we want en-us be part of the URL since it breaks automatic locale logic on the learn/docs pages (see #151 )

Author: Thanks, good catch!

> [Windows UI Library (WinUI)](https://ocs.microsoft.com/windows/apps/winui)  

Reviewer: ```suggestion
[WinUI](https://aka.ms/windev)  

```

- **Design & Accessibility Guidance**: Design and accessibility pages help make the gallery a useful developer companion app. 
- **Version Adaptive Code:** This sample shows how to write version adaptive code so that the app can run on previous versions of Windows while also using the latest capabilities on the most recent verison of Windows.

## 👏 Contribute to WinUI Gallery

Any samples or docs improvements you'd like to see? We're always looking for a helping hand. Feel free to file an issue to start the discussion, or even better, create a PR with the change you'd like to see!

> Check out these [help wanted](https://github.com/microsoft/WinUI-Gallery/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) issues if you don't know where to start.

Reviewer 2: What about the "good first issue" label? Should we support both or should we stick with one?

Author: What do you think? both help wanted and good first issue have a lot of overlap. I wonder if it makes sense to just consolidate to one. WinUI Gallery repo is also a lot less complicated than the WinUI2 repo, so a "good first issue" may not be needed?

Reviewer 2: I agree with you, for larger repositories, good first issue might be a good idea. In our case, help wanted is probably enough.

**Required [Visual Studio components](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/set-up-your-development-environment?tabs=cs-vs-community%2Ccpp-vs-community%2Cvs-2022-17-1-a%2Cvs-2022-17-1-b#required-workloads-and-components):**
- .NET Desktop Development.
- Windows App SDK C# Templates
- Windows 10 SDK (10.0.19041.0)

### 2. Clone the Repository

```shell
git clone https://github.com/microsoft/WinUI-Gallery.git
```

### 3. Open .sln with Visual Studio and Build!
```shell
.\WinUIGallery\WinUIGallery.DesktopWap.sln
```
Make sure to select `WinUIGallery.DesktopWap.Package` for startup project in Visual Studio.

### Further information

To obtain information about Windows 11 development, go to the [Windows Dev Center](https://developer.microsoft.com/windows)

To obtain information about Microsoft Visual Studio and the tools for developing Windows apps, go to [Visual Studio](http://go.microsoft.com/fwlink/?LinkID=532422)

To contact the authors, please reach out to ControlsGallery@microsoft.com

## 🏆 Contributors
Thanks to our amazing contributors!

> [![Toolkit Contributors](https://contrib.rocks/image?repo=microsoft/WinUI-Gallery)](https://github.com/microsoft/WinUI-Gallery/graphs/contributors)

Reviewer 2: I don't think those are toolkit contributors :)

Author: Good catch!
